.github/workflows/test.yml

```
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        network: [testnet, pubnet, standalone]
+        network: [testnet, pubnet]
         dockerfile: [Dockerfile, Dockerfile.testing]
```

❗ Instead of adding the `test-standalone` section as an entirely separate job to test captive core only with standalone, you can add a specific slice of a matrix to the matrix using the `include` keyword.

i.e.:

```suggestion
        network: [testnet, pubnet, standalone]
        dockerfile: [Dockerfile, Dockerfile.testing]
        include:
        - network: standalone
          options: ["", "--enable-horizon-captive-core"]
```

If I haven't got the syntax quite right, see https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix.